### ISSUE TITLE
Cannot install with latest react-native 0.64.2

### ISSUE THREAD

**Issue by codemem** [jaredh159/tailwind-react-native-classnames#20]
```js
npm ERR! Found: react-native@0.64.2
npm ERR! node_modules/react-native
npm ERR!   react-native@"0.64.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react-native@"^0.63.0" from tailwind-react-native-classnames@1.4.0
npm ERR! node_modules/tailwind-react-native-classnames
npm ERR!   tailwind-react-native-classnames@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by danielolamide** [comment]
I added the legacy-peer-deps flag and so far its working well `npm install tailwind-react-native-classnames --legacy-peer-deps`

**Comment by jaredh159** [comment]
just released `1.4.1` which relaxes the peer dep to allow for RN `0.64.x`. Thanks for bringing this to my attention.
